src/var/intervalSet.js

```diff
@@ -0,0 +1,4 @@
+define(function() {
```

Comment:
You may notice I did it correctly in other places, though. ;) Here I just copied a template from another file, hence the mistake.

Once JSCS starts checking this, the problem will go away. ;)